fe1-web/model/network/method/message/data/election/CreateElection.ts

```diff
@@ -0,0 +1,94 @@
+
+  public readonly version: string;
+
+  public readonly created_at: Timestamp;
+
+  public readonly start_time: Timestamp;
+
+  public readonly end_time: Timestamp;
+
+  public readonly questions: {
+    id: string,
+    question: string,
+    voting_method: string,
+    ballot_options: string[],
+    write_in: boolean,
+  }[];
+
+  constructor(msg: Partial<CreateElection>) {
+    if (!msg.id) throw new ProtocolError('Undefined \'id\' parameter encountered during \'CreateElection\'');
+    this.id = msg.id;
+
+    if (!msg.name) throw new ProtocolError('Undefined \'name\' parameter encountered during \'CreateElection\'');
+    this.name = msg.name;
+
+    if (!msg.version) throw new ProtocolError('Undefined \'version\' parameter encountered during \'CreateElection\'');
+    this.version = msg.version;
+
+    if (!msg.created_at) throw new ProtocolError('Undefined \'created_at\' parameter encountered during \'CreateElection\'');
+    checkTimestampStaleness(msg.created_at);
+    this.created_at = msg.created_at;
+
+    if (!msg.start_time) throw new ProtocolError('Undefined \'start_time\' parameter encountered during \'CreateElection\'');
+    checkTimestampStaleness(msg.start_time);
+    this.start_time = msg.start_time;
+
+    if (!msg.end_time) throw new ProtocolError('Undefined \'end_time\' parameter encountered during \'CreateElection\'');
+    checkTimestampStaleness(msg.end_time);
+    if (msg.end_time < msg.created_at) throw new ProtocolError('Invalid timestamp encountered: \'end\' parameter smaller than \'creation\'');
+    this.end_time = msg.end_time;
+
+    if (!msg.questions) throw new ProtocolError('Undefined \'questions\' parameter encountered during \'CreateElection\'');
+    msg.questions.forEach((question) => {
+      if (!question.id) throw new ProtocolError('Undefined \'question id\' parameter encountered during \'CreateElection\'');
+      if (!question.question) throw new ProtocolError('Undefined \'question\' parameter encountered during \'CreateElection\'');
+      if (!question.voting_method) throw new ProtocolError('Undefined \'voting method\' parameter encountered during \'CreateElection\'');
+      if (!question.ballot_options) throw new ProtocolError('Undefined \'ballot_options\' parameter encountered during \'CreateElection\'');
+    });
+    this.questions = msg.questions;
+
+    // FIXME: implementation not finished, get event from storage,
+    // eslint-disable-next-line @typescript-eslint/no-unused-vars
+    const lao: Lao = OpenedLaoStore.get();
+    console.log('Create Election called: ', this);
+
+    // const expectedHash = Hash.fromStringArray(
+    //   EventTags.ELECTION, lao.id.toString(), lao.creation.toString(), msg.name,
+    // );
+    // eslint-disable-next-line max-len
+    // if (!expectedHash.equals(msg.id)) throw new ProtocolError('Invalid \'id\' parameter encountered during \'CreateMeeting\': unexpected id value');
+  }
+
+  public static fromJson(obj: any): CreateElection {
```

Comment:
you might want to add tests for this, just to make sure that it works properly :)